docs/security.html

```diff
@@ -398,7 +398,7 @@ <h5>Host Name Verification</h5>
             ssl.keystore.password=test1234
             ssl.key.password=test1234</pre>
 
-            Other configuration settings that may also be needed depending on our requirements and the broker configuration:
+            Other configuration settings that may also be needed depending on requirements and the broker configuration:
```

Comment:
Remove the redundant `our` here

Reply:
I'm not sure this is redundant. This refers to the user's requirements.